controllers/sessions.js

```diff
@@ -1,71 +1,78 @@
-const fs = require('fs');
-const Sessions = require('../data/sessions.json');
-
-const listSession = (req, res) => {
-  res.status(200).json(Sessions);
-};
+const Sessions = require('../models/Sessions');
 
 const createSession = (req, res) => {
-  const newSession = {
-    idSession: req.query.idSession,
-    idPsicologist: req.query.idPsicologist,
-    idPostulant: req.query.idPostulant,
-    date: req.query.date,
-    state: req.query.state,
-  };
-  Sessions.push(newSession);
-  fs.writeFile('./data/sessions.json', JSON.stringify(Sessions), {}, (error) => {
-    if (error) {
-      return res.status(400).send(error);
+  const session = new Sessions({
+    postulant: req.body.postulant,
+    psychologist: req.body.psychologist,
+    status: req.body.status,
+    date: req.body.date,
+    notes: req.body.notes,
+  });
+  session.save((err, cbSession) => {
+    if (err) {
+      return res.status(400).json(err);
     }
-    return res.status(201).json(newSession);
+    return res.status(201).json(cbSession);
   });
 };
 
 const updateSession = (req, res) => {
-  let updatedSession;
-  const updatedSessions = Sessions.map((session) => {
-    if (session.idSession === req.query.idSession) {
-      updatedSession = {
-        idSession: req.query.idSession,
-        idPsicologist: req.query.idPsicologist,
-        idPostulant: req.query.idPostulant,
-        date: req.query.date || session.date,
-        state: req.query.state || session.state,
-      };
-      return updatedSession;
+  Sessions.findByIdAndUpdate(
+    req.params.id,
+    {
+      postulant: req.body.postulant,
+      psychologist: req.body.psychologist,
+      status: req.body.status,
+      date: req.body.date,
+      notes: req.body.notes,
+    },
+    { new: true },
+    (err, newSession) => {
+      if (err) {
+        return res.status(400).json(err);
+      }
+      if (!newSession) {
+        return res.status(404).json({ msg: `The session 'id' (${req.params.id}) given  does not exist.` });
+      }
+      return res.status(200).json(newSession);
+    },
+  );
+};
+
+const deleteSession = (req, res) => {
+  Sessions.findByIdAndDelete(req.params.id, (err, deletedSession) => {
+    if (err) {
+      return res.status(400).json(err);
     }
-    return session;
-  });
-  if (!updatedSession) res.status(404).send('Session NOT found');
-  fs.writeFile('./data/sessions.json', JSON.stringify(updatedSessions), {}, (error) => {
-    if (error) {
-      return res.status(400).send(error);
+    if (!deletedSession) {
+      return res.status(404).json({ msg: `The session 'id' (${req.params.id}) given  does not exist.` });
     }
-    return res.status(200).json(updatedSession);
+    return res.status(204).send(deletedSession);
```

Comment:
204 not allow data response, just send a message

Reply:
not even a message, just write 
```
return res.status(204).send()
```